src/Themes/DigitalTheme/Views/SharedLoginFooter.liquid

```diff
@@ -0,0 +1,3 @@
+        </section>
```

Comment:
We should experiment with shape wrappers instead of having partial html tags in different files.

Reply:
I had looked into this a few months ago and couldn't get the shape wrapper to work the way I want it to work.